README.md

```diff
@@ -43,6 +43,42 @@ The Initialize methods are automatically called by the GameObjectExtention metho
 
 However you can just add your class that implements `IHierarchyBehaviour` and choose to call Initialize when you prefer to.
 
+In addition you can also implement multiple `IHierarchyBehaviour`'s, for example:
+```c#
+public class LightEstimation : MonoBehaviour, IHierarchyBehaviour, IHierarchyBehaviour<Light[]>
+{
+    private Light[] lights;
+
+    public void Initialize()
+    {
+        lights = FindObjectsOfType<Light>();
+    }
+    
+    public void Initialize(params Light[] lights)
```

Comment:
How does params work with interfaces. Since the interface is `Initialize(T)` I'm guessing if I have reference to the interface it will jut let me put the specific type in and not let me do `Initialize(arg, arg, arg)`. But if you have a reference to the `LightEstimation` I can probably take advantage of it

Reply:
You can only do params when its directly called on it and not via any form that uses generics